Build/Utility.targets

```diff
@@ -109,9 +109,13 @@ Copyright 2017 Microsoft. All rights reserved.
   <Target
     Name="GeneratePublicationManifest"
     DependsOnTargets="Xamarin_Build_ReadAllProperties">
-    <PropertyGroup>
-      <UpdateInfoFile Condition="'$(OS)' == 'Windows_NT'">..\Package\Windows\updateinfo</UpdateInfoFile>
-      <UpdateInfoFile Condition="'$(OS)' == 'Unix'">..\Package\Mac\updateinfo</UpdateInfoFile>
+    <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
+      <ProductName>Xamarin Workbooks &amp; Inspector (Win)</ProductName>
+      <UpdateInfoFile>..\Package\Windows\updateinfo</UpdateInfoFile>
+    </PropertyGroup>
+    <PropertyGroup Condition="'$(OS)' == 'Unix'">
+      <ProductName>Xamarin Workbooks &amp; Inspector (Mac)</ProductName>
```

Comment:
Can we start changing this to just "Xamarin Workbooks" right here, or do we need to coordinate with other people?

We probably want to stop advertising ourselves as "Inspector" everywhere starting with 15.6.

Reply:
We cannot. Certainly not in this PR.